client/src/parserFunctions.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+const parserFunctions = {
+  getRelatedItems: (productId, cachedProducts) => {
+    return axios.get(`/products/${productId}/related`)
+      .then(({data}) => {
+        let relatedProducts = data.map(product => {
+          if (cachedProducts[product]) {
+            return ['find'];
```

Comment:
Maybe you could help me understand what this helper function is doing and what its' expected to return

Reply:
This is a bug. I was testing this function and forgot to change it.
Thanks for catching that.